gnocchi/tests/functional/gabbits/batch-measures.yaml

```diff
@@ -188,6 +216,25 @@ tests:
       response_json_paths:
         $:
           - ["2015-03-06T14:34:12+00:00", 1.0, 12.0]
+          - ["2015-03-06T14:35:12+00:00", 1.0, 24.0]
+
+    - name: get created metric to check creation with newformat
+      GET: /v1/resource/generic/46c9418d-d63b-4cdd-be89-8f57ffc5952e/metric/newformat
```

Comment:
should you check that this newformat uses the defined policy and not a default policy.

Reply:
This is checked line 228

Reply:
word :)